src/GuessControl.jsx

```diff
@@ -1,47 +1,29 @@
-import React, { Component } from "react";
+import { useState } from "react";
 import Button from "./Button";
 
-class GuessControl extends Component {
-  constructor(props) {
-    super(props);
+function GuessControl({ onGuess }) {
+  const [currentGuess, setCurrentGuess] = useState("");
 
-    this.state = {
-      currentGuess: "",
-    };
-
-    /**
-     * These lines are required to make the methods/functions declared on this
-     *  class have the correct `this` object when they run.
-     */
-    this.handleInputChange = this.handleInputChange.bind(this);
-    this.onSubmitGuess = this.onSubmitGuess.bind(this);
-  }
-
-  handleInputChange(event) {
-    this.setState({ currentGuess: event.target.value });
+  const handleInputChange = (event) => {
+    event.preventDefault();
```

Comment:
do you need an `event.preventDefault()` here?

Reply:
the line removed